lareventdisplay/EventDisplay/SimulationDrawer.cxx

```diff
@@ -653,7 +653,7 @@ namespace evd {
     // Use the LArVoxelList to get the true energy deposition locations as
     // opposed to using MCTrajectories
     const sim::LArVoxelList voxels =
-      sim::SimListUtils::GetLArVoxelList(evt, drawopt->fG4ModuleLabel.label());
+      sim::SimListUtils::GetLArVoxelList(evt, drawopt->fSimChannelLabel.encode());
```

Comment:
Having to call `encode()` is icky, but there's not much you can do about that here.  Do you mind putting some type of comment that says "GetLArVoxelList should be adjusted to take an `InputTag` instead of string"?  Then people know  why this hack has to be done here.

Reply:
Thanks. I just added the comment. Yes, I agree that function should be adjusted. But I am also worried other code may be using that function also.